system/Test/FeatureTestTrait.php

```diff
@@ -165,6 +178,7 @@ public function call(string $method, string $path, array $params = null)
 		$request = $this->setupRequest($method, $path);
 		$request = $this->setupHeaders($request);
 		$request = $this->populateGlobals($method, $request, $params);
+		$request = $this->setRequestBody($request);
```

Comment:
I'm curious about this addition. Why was this necessary? What happens without it?

Reply:
That's the method that actually does the work of formatting the parameters and putting them on the body.